spec/features/course/duplication_spec.rb

```diff
@@ -58,17 +58,14 @@
         end
 
         scenario 'I can duplicate objects from that course' do
-          visit course_duplication_path(course)
-
-          find('.source-course-dropdown').click
-          find("[role='option']", text: source_course.title).click
+          visit course_duplication_path(source_course)
 
           find("input[value='OBJECT']", visible: false).click
 
-          find('.destination-course-dropdown').click
+          find("[class*='destination-course-dropdown']").click
```

Comment:
What's the reason for changing the syntax from css class selector `.destination-course-dropdown` to this?

Reply:
we're migrating from jsx to tsx and in tsx, we're mostly using className for styling and also component naming.. In here, the previous syntax didn't work (they couldn't find `.destination-course-dropdown` anymore since it's embedded inside the class). Hence the change of the syntax

Reply:
it's weird.. I reverted it back (together with the other line, which also was used to be `find('.items-selector-(something)')` and it works now... I have reverted it back as per requested